python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_graph/_digraph_group_chat.py

```diff
@@ -469,11 +465,21 @@ async def select_speaker(self, thread: Sequence[BaseAgentEvent | BaseChatMessage
             # Reset the bookkeeping for the specific activation groups that were triggered
             self._reset_triggered_activation_groups(speaker)
 
-        # If there are no speakers, trigger the stop agent.
+        # If there are no speakers, the graph execution is complete.
```

Comment:
GraphFlowManager should override BaseGroupChatManager._apply_termination_condition instead of using `select_spaker` for performing termination procedure.